src/modules/mongo.js

```diff
@@ -1,12 +1,21 @@
 import dotenv from 'dotenv';
+import mongoose from 'mongoose';
 import { logger } from './globals.js';
 
 const connectToDB = () => {
```

Comment:
The connection is async so the block should be as well.
```suggestion
const connectToDB = async () => {
```